aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -283,6 +283,9 @@ void unary_op(const Tensor& self, const Tensor& output, std::string op_name, Una
 {
     using namespace mps;
```

Comment:
Does this need an assert or a log ? Is this expected to be 0 sometimes ?

Reply:
Yes, it's expected, and it's not an error. Sometimes the previous op may not yield any results but will pass the empty result to the next op which is fine. This check exists in most ops in PyTorch.